lisa/sut_orchestrator/azure/common.py

```diff
@@ -60,22 +60,38 @@ class AzureVmMarketplaceSchema:
     version: str = "Latest"
 
 
+@dataclass_json()
```

Comment:
Because it doesn't need to be serializable, so the dataclass_json is not necessary.

Reply:
This would also be applicable to `AzureVmMarketplaceSchema`, right?